app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str):
+    file1 = command.split(" ")[1]
+    file2 = command.split(" ")[2]
+    if file1 != file2:
```

Comment:
also, you should check the input command

Reply:
What exactly needs to be checked in the input command?

Reply:
<img width="839" alt="image" src="https://user-images.githubusercontent.com/72562232/184335196-afa79928-a551-4b02-a76d-1c912706819b.png">